pages/docs/guides/validate.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Validate AsyncAPI Documents"
+description: In this guide, you'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+# Introduction
+When people talk about AsyncAPI validation, sometimes they mean completely different things: **validation of AsyncAPI documents** or **validation of messages in runtime** _(message validation against schemas provided in the AsyncAPI document)_. 
+
+This guide is about validating AsyncAPI documents to make sure they are correct. 
+
+# Create AsyncAPI documents
+To validate an AsyncAPI document, you first need to have one. If you did not create one for your app yet, you can easily do so by using the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+# Validate AsyncAPI documents
+Validating an AsyncAPI document can mean one of two things: 
+1. Validation against the specification.
+2. Validation against the best practices or company governance rules.
+
+## Validate against specification
+Let's discuss options for validating against the specification. 
+
+This ensures that every content of the document is written in accordance with the AsyncAPI specification.
+
+There are several tool options you may select for validating against the specification: _AsyncAPI Studio_, _AsyncAPI CLI_, and _Parsers_.
+
+### AsyncAPI Studio validation 
+[AsyncAPI Studio](https://github.com/asyncapi/studio#readme) provides a visual and easy way to validate your AsyncAPI documents against the specification. 
+
+It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.
+
+Errors in your document are highlighted with a red underline, showing which lines are invalid. The console also provides feedback, allowing you to further troubleshoot with detailed error messages.
+
+When a document is invalid, it provides the following error: `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+### AsyncAPI CLI validation 
+The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in automation:
+
+ ```
+ asyncapi validate asyncapi.yaml
+
+ ```
+
+> You can also open the AsyncAPI Studio from the CLI by running the command `asyncapi start studio`
```

Comment:
```suggestion
<Remember>
You can also open AsyncAPI Studio from the CLI by running the command `asyncapi start studio`.
</Remember>
```

When adding an important note, please use our React component for `Remember` boxes. 😄 That is a better UX and more easily seen than this markdown note.